Makefile

```diff
@@ -25,7 +25,7 @@ else
 endif
 
 EXTENSION = pg_shard
-DATA = pg_shard--1.0.sql
+DATA = pg_shard--1.1.sql pg_shard--1.0--1.1.sql
```

Comment:
The only way to install version 1.0 would be to download 1.0 and use that. The state of affairs for our extensions seems to be:

  * The latest download contains a script to install that latest version
  * It also contains upgrade scripts from earlier versions